Task3_Position_the_Cubes/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        *{
+            box-sizing: border-box;
+            margin: 0px;
+            padding: 0px;
+        }
+        .c1{
+         border: 2px solid white;
+         background-color: red;
+         height: 100px;
+         width: 100px;
+         position: relative;
+         top: 0px;
+         left: 0px;
+        } 

```

Comment:
Consider adding a comment to explain the purpose of this CSS block for better code documentation